packages/wrangler/src/config/index.ts

```diff
@@ -49,7 +49,7 @@ export function readConfig<CommandArgs>(
 		args
 	);
 
-	if (diagnostics.hasWarnings()) {
+	if (diagnostics.hasWarnings() && !args?.json) {
```

Comment:
@petebacondarwin - I could use your help here, I'm trying to avoid adding `args.json` to the `CommonYargsOptions` type, and it should be there in `CommandArgs`, but tsc complains:

```
Property 'json' does not exist on type 'CommandArgs & OnlyCamelCase<CommonYargsOptions>'.ts(2339)
```